README.markdown

```diff
@@ -4,89 +4,50 @@
+5. [Reference](#reference)
+    * [Public Defined Types](#public-defined-types)
+    * [Public Custom Types](#public-custom-types)
+    * [Parameters](#parameters)
+6. [Limitations](#limitations)
+7. [Development - Guide for contributing to registry](#development)
 
 ##Overview
 
-The registry module enables you to manage your Windows Registry from your *nix puppet master by supplying the types and providers necessary to create and manage Windows Registry keys and values with Puppet. 
-
-##Setup
-
-###What registry affects
-
-* files, folders, and services in Windows Registry    
-* node manifests
+This module supplies the types and providers you'll need to manage the Registry on your Windows nodes.
 
-###Setup requirements
+##Module Description
 
-For this module to run correctly, you must have [pluginsync enabled](http://docs.puppetlabs.com/guides/plugins_in_modules.html#enabling-pluginsync). 
+The Registry is a hierarchical database built into Microsoft Windows. It stores settings and other information for the operating system and a wide range of applications. This module lets Puppet manage individual Registry keys and values, and provides a simplified way to manage Windows services.
 
-This setting will ensure the types and providers are synchronized and available on the agent before the configuration run takes place. This is the default behavior of the puppet agent on Microsoft Windows platforms. 
-
-###Beginning with Registry
-
-The registry module is intended to enable a *nix master server to interact with a Windows agent. The module must be downloaded and installed on your puppet master.
+##Setup
 
-The bulk of registry's capabilities comes from two resource types: `registry_key` and `registry_value`. Combined, these types allow you to specify the Registry container and the file(s) meant to be in it. 
+This module must be installed on your Puppet master. We've tested it with Puppet agents running on Windows Server 2003, 2008 R2, 2012, and 2012 R2.
 
-The defined resource type `registry::value` allows you to manage Registry values and the parent key in one fell swoop.
+###Beginning with registry
 
-    registry::value { 'MyApp Setting1':
-      key   => 'HKLM\Software\Vendor\PuppetLabs',
-      value => setting1,
-      data  => 'Hello World!'
-    }
-    
-Within this defined type, you can specify multiple Registry values for one registry key and manage both at once. 
-
-If, on the other hand, you just want to manage a standalone key or value, you can declare each resource type individually in your site.pp manifest.
+Use the `registry_key` type to manage a single registry key:
 
     registry_key { 'HKLM\System\CurrentControlSet\Services\Puppet':
       ensure => present,
     }
 
-and/or
+##Usage
+
+The registry module works mainly through two custom resource types: `registry_key` and `registry_value`. These types combine to let you specify a Registry container and its intended contents.
```

Comment:
This represents a global shift we recently decided to make for all new docs. Despite our own reference materials, "define" is already the term most commonly used by the user community. Existing documentation will be updated over time.